client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -42,11 +117,55 @@ export default {
 
   components: {
     FieldPicker,
+    VueSelect,
   },
 
   extends: base,
+
+  computed: {
+    documentationURL () {
+      // eslint-disable-next-line no-undef
+      const [year, month] = VERSION.split('.')
+      return `https://docs.cortezaproject.org/corteza-docs/${year}.${month}/integrator-guide/compose-configuration/index.html`
+    },
+
+    fieldOptions () {
+      const nonRequiredFields = this.module.fields.filter(f => !f.isRequired)
+      const unConditionedFields = nonRequiredFields.filter(f => !this.options.fieldConditions.find(fc => fc.field.name === f.name))
+      const fields = unConditionedFields.map(f => {
+        return {
+          name: f.name,
+          label: f.label,
+          fieldID: f.fieldID,
+          isRequired: f.isRequired,
+        }
+      })
+      return fields
+    },
+  },
+
+  created () {
+    console.log(this.fieldOptions)
+  },
+
+  methods: {
+    add () {
+      if (this.options.fieldConditions.length >= this.module.fields.length) return
```

Comment:
No need for this check, you won't be able to select a field anyhow, since the selectable prop will disable that

Reply:
I think the user should not be making field conditions even after it is clear that they have run out of fields.

Reply:
No need for that. Even if you wanted to do that you would disable the button that adds a new rule and not prevent it from being added in the function itself